test/sort-enums.test.ts

```diff
@@ -1977,12 +1977,15 @@ describe(ruleName, () => {
       )
     })
 
-    describe('handles complex comment cases', () => {
-      ruleTester.run(`keeps comments associated to their node`, rule, {
-        valid: [],
-        invalid: [
-          {
-            code: dedent`
+    describe(`${ruleName}: handles complex comment cases`, () => {
```

Comment:
No actual change in the tests here, just added `${ruleName}:` which re-indented following lines.